src/main/java/jalau/at18/architects/ResourceCounter.java

```diff
@@ -0,0 +1,38 @@
+package jalau.at18.architects;
+
+import java.util.LinkedList;
+
+public class ResourceCounter {
+    private LinkedList<String> listOfResources;
+    private GrayCard grayCard;
+
+    public ResourceCounter(Resource resource, LinkedList<String> listOfResources) {
+        this.grayCard = new GrayCard(resource);
+        this.listOfResources = listOfResources;
+    }
+
+    public LinkedList<String> getListOfResources() {
+        return listOfResources;
+    }
+
+    public void addResourceToList() {
+        listOfResources.add(grayCard.getResource());
+    }
+
+    public boolean traverseTheResourceList() {
+        int counter = 0;
+        for (String element : listOfResources) {
+            boolean sameValue = (listOfResources.get(counter + 1) == element) ? true : false;
```

Comment:
Not sure if a list is really needed here, since you compare two resources and tell if those are equal, you could simply have those two resources without a list. Why a list would be needed?

Reply:
Jose in this case, we are using the list because we are planning to remove the elements from them and once the elements are compared, we will proceed to eliminate those elements with remove() method

Reply:
So all the elements that match a certain card would be removed?

Reply:
Yes exactly, but now I decided to change it to only compare resources, we still have to work on the condition to build a Wonder stage